cooler_manipulation.ipynb

```diff
@@ -0,0 +1,1524 @@
+{
```

Comment:
<p>maybe be more specific with what is done: Here we show an example where two chromosomes are split into 4 arms, re-orderd, and inverted.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
if before/after could be put in side-by-side subplot and have titles (e.g. original, re-orderd) this would be a bit more clear